plugins/woocommerce-admin/client/tasks/task-list-item.tsx

```diff
@@ -130,16 +126,15 @@ export const TaskListItem: React.FC< TaskListItemProps > = ( {
 
 	const onClickDefault = useCallback( () => {
 		if ( actionUrl ) {
-			if ( actionUrl.startsWith( 'http' ) ) {
-				window.location.href = actionUrl;
-			} else {
-				getHistory().push( getNewPath( {}, actionUrl, {} ) );
-			}
+			navigateTo( {
```

Comment:
The issue here is that we currently have `actionUrl`'s of simply `/setup-wizard`, as well as full absolute URLs (as a side note we might want to revisit the format of the `actionUrl` for this reason, or at least document it somewhere clearly). Since this is a bit of a niche/non-standard format, it seemed cleaner to keep this logic here. I'm currently thinking that the cleanest implementation of `navigateTo` only accepts an absolute URL, or one relative to the admin directory, which works well with our existing utilities like `getNewPath()`. 

We _could_ just assume that anything that isn't absolute and does not start with a php file is a wcadmin path, perhaps? I'm reluctant to start adding a bunch of regex and various conditions to try to parse several different potential formats, but that may be a good middle-ground. That's what I've now done in https://github.com/woocommerce/woocommerce/pull/33124/commits/dfd9f77b55fcb3c7144ede9e8574a5f86970df5c. Let me know what you think @joshuatf

Reply:
Ya, that makes sense.  We really should not have introduced the pattern of using partial WCA URLs as this is coming back to bite us now.

But I like the short term regex check.  This feels like a good intermediary until we can resolve the bad partial URLs.